python/neuroglancer/default_credentials_manager.py

```diff
@@ -26,3 +26,7 @@
 default_credentials_manager.register(
     u'gcs',
     lambda _parameters: google_credentials.get_google_application_default_credentials_provider())
+
+default_credentials_manager.register(
+    u'DVID',
+    lambda _parameters: dvid_credentials.get_tokenbased_application_default_credentials_provider(_parameters))
```

Comment:
since `_parameters` is now used, you can rename it to `parameters`   The underscore is just to silence lint warnings about unused variables.

Reply:
Done!